packages/f/freetype/xmake.lua

```diff
@@ -10,41 +10,54 @@ package("freetype")
     add_versions("2.10.4", "5eab795ebb23ac77001cfb68b7d4d50b5d6c7469247b0b01b2c953269f658dac")
     add_versions("2.9.1", "ec391504e55498adceb30baceebd147a6e963f636eb617424bcfc47a169898ce")
 
-    local configdeps = {woff2 = "brotli",
-                        bzip2 = "bzip2",
-                        png   = "libpng",
-                        zlib  = "zlib"}
+    add_extsources("apt::libfreetype6", "pkgconfig::freetype2", "brew::freetype")
```

Comment:
if is_plat("macosx") then
    add_extsources("brew::freetype")
end

Reply:
why do we have to check the platform before adding a extsource? xmake won't search for brew packages if brew isn't available as far as i know

Reply:
it will find too brew, this is not necessary and will be added to the list of manually installed third-party packages